assessments/projects/linkerd/self-assessment.md

```diff
@@ -0,0 +1,321 @@
+# Self-assessment
+The Self-assessment is the initial document for projects to begin thinking about the
```

Comment:
Please go back to ensure that no template text like this is included here or in the rest of the document